### ISSUE TITLE
Vim mode: incorrect behavior of <shift + key> for different characters

### ISSUE THREAD

**Issue by serpro69** [laurent22/joplin#4280]
Using `f` or `t` for line forward search does not work with some upper case chars.

The backward search, at the same time, works fine it seems.

This started happening when I upgraded to 1.5.14. Though I'm not sure what version it was introduced in as I haven't upgraded for some months. I also tried to go back version by version and went as far as 1.1.xxx and the issue was still there, though I'm pretty sure it worked fine before.

## Environment

Joplin 1.5.14 (prod, linux)
Linux 5.4.0-58-generic 
Ubuntu 20.04

## Steps to reproduce

1. Enter vim mode
2. Type sample text in one line  `Lorem ipsum dolor sit amet, A consectetur adipiscing elit, I sed do eiusmod tempor incididunt ut labore O et dolore magna aliqua.`
3. Go to beginning of the line.
4. Jump to first occurrence of A / I / O with `f <shift+ a / i / o >`

## Describe what you expected to happen

**Expected**: cursor should jump to A / I / O

**Actual**: the action of `<shift + a / i / o>` is executed instead. For example `<shift + a>` will move the cursor to the end of line and switch to insert mode, `<shift + o>` will insert empty line above and switch to insert mode.

UPD:
The above is just an example, it looks like there are more issues where `<shift + ... >` is used incorrectly. For example replacing a char with `{` or `}` does not work and instead of replacement the cursor jumps to the next/previous paragraph.

**Comment by CalebJohn** [comment]
I can't reproduce this issue on my system (identical to yours). Have you tried restarting Joplin to see if it still occurs?

In any case, this is an upstream bug with the editor, and will needed to be [reported there](https://github.com/codemirror/CodeMirror) if you can reproduce it.

**Comment by serpro69** [comment]
Hi @CalebJohn , 
Yes, I have tried restarting Joplin and tried a bunch of previous versions as well - didn't help. I think it started after migration to codemirror happened (I must have been on an ancient version of Joplin before upgrading to 1.5.14) since I haven't experienced this before. I'll try to reproduce it in codemirror and report it there. 
Thanks

**Comment by serpro69** [comment]
So I've tried to reproduce this in the [codemirror vim demo page](https://codemirror.net/demo/vim.html).

I'm using Brave Browser as default browser and I can reproduce the issue there.
I've also tried Firefox, and there it works just fine.

Any ideas how that correlates to an issue within Joplin, @CalebJohn ?

**Comment by CalebJohn** [comment]
If I'm correct in understanding you, the issue persists when you use the demo on Brave browser but not on Firefox?

In that case you'll want to [report the bug to CodeMirror directly](https://github.com/codemirror/codemirror) because the issue is with the CodeMirror editor (that Joplin uses) and not with Joplin itself.

**Comment by serpro69** [comment]
Closing as this is not an issue with joplin.
Just in case someone stumbles upon the same. I've found that the issue was with how Ubuntu was using CAPS key, which was causing issues. Strange thing is, the bug wasn't reproducible in FIirefox, but was in Brave, though it should have been since it was an issue in my system, not with a particular browser. So in case anyone's facing the same - check your keyboard bindings, and test out things in the codemirror vim demo page.
